tests/features/unittest_aliases_test.py

```diff
@@ -64,6 +69,21 @@ def test_fix_unittest_aliases(s, expected):
             'unittest.getTestCaseNames(MyTests)',
             'unittest.defaultTestLoader.getTestCaseNames(MyTests)',
         ),
+        (
+            'unittest.findTestCases(MyTests, kw="arg")',
+            'unittest.defaultTestLoader.loadTestsFromModule' +
+            '(MyTests, kw="arg")',
+        ),
+        (
+            'unittest.makeSuite(MyTests, kw="arg")',
+            'unittest.defaultTestLoader.loadTestsFromTestCase' +
+            '(MyTests, kw="arg")',
+        ),
+        (
+            'unittest.getTestCaseNames(MyTests, kw="arg")',
+            'unittest.defaultTestLoader.getTestCaseNames' +
+            '(MyTests, kw="arg")',
+        ),
```

Comment:
I think these are broken too -- the only valid rewrite case is exactly 1 positional argument

Reply:
Sorry, my mistake. Yes, that's the only valid scenario. I updated the logic and the test cases.